src/gridfs/file.rs

```diff
@@ -266,6 +273,7 @@ impl File {
             let result = arc_gfs.chunks.insert_one(document, None);
 
             // Complete pending write
+            // XXX: shouldn't this be `_variable` too? Assigning to `_` drops the guard immediately.
```

Comment:
yes, the fact that we call `cvar.notify_all()` at the end of the block here, and condvar is only used to wait on the guard/error description indicates that the lock is supposed to be kept here.

Reply:
Sure, I'll put in a variable for this too, then.